api-reference/openapi.yaml

```diff
@@ -2735,3 +2735,159 @@ components:
             self:
               type: string
               example: https://api.ctrl-hub.com/v3/governance/schemes/5852cf16-8ec8-4aba-809e-f8d828772c4e/work-orders/5852cf16-8ec8-4aba-809e-f8d828772c4e/permits/5852cf16-8ec8-4aba-809e-f8d828772c4e
+    dataCaptureFormSubmissionVersionModel:
+      type: object
+      properties:
+        type:
+          type: string
+          example: versions
```

Comment:
Should this be `submission-versions` to disambiguate with things like form versions?